pkg/secrets/secrets.go

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+package secrets
+
+import (
+    "strings"
+
+    galasaErrors "github.com/galasa-dev/cli/pkg/errors"
+)
+
+func validateSecretName(secretName string) error {
+    var err error
+    secretName = strings.TrimSpace(secretName)
+
+    if secretName == "" || strings.ContainsAny(secretName, " \n\t") {
```

Comment:
I should think we need to validate it more than that ? 
eg: "a..secret....with.lots.of.dots" will blow up etcd no ?
Latin-1 characters only ? what about - and _ are they allowed ?